package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -678,6 +678,43 @@ def analysis(current, output, u, **kwargs):
       :attr:`~WaterBridgeAnalysis.timeseries` to find the specific time point
       of a water bridge existence.
 
+   .. attribute:: results.network
+
+      Network representation of the water network.
+
+      .. versionadded:: 2.0.0
+
+   .. attribute:: network
+
+      Alias to the :attr:`results.network` attribute.
+
+      .. deprecated:: 2.0.0
+         Will be removed in MDAnalysis 3.0.0. Please use
+         :attr:`results.hbonds` instead.
+
+   .. attribute:: _network
```

Comment:
I think one doesn't need to document a private variable.

Reply:
Fair enough, then we also do not need to have a property alias for it.

Reply:
Good point. I was initially a bit confused because the `_network` was the attribute actually being used in the tests, therefore I thought it was something users could use as well (I often look at tests to see how things are used). Since it is private and there is a public counterpart, I will just remove this and deprecate `network` in favor of `results.network`.